tests/testthat/test-doy_noy.R

```diff
@@ -1,3 +1,28 @@
+vp <- example_vp
+vpts <- example_vpts
+vpi <- integrate_profile(example_vpts)
+pvolfile <- system.file("extdata", "volume.h5", package = "bioRad")
+pvol <- read_pvolfile(pvolfile)
+
 test_that("doy_noy() returns error on incorrect parameters", {
+  expect_error(doy("not_a_object"))
+  expect_error(noy("not_a_object"))
+
+})
+
+test_that("doy_noy() returns a number", {
+  expect_true(is.numeric(doy(vp)))
```

Comment:
it is better to use  `expect_type()` to test data types. So here:

    expect_type(doy(vp), "integer")

Reply:
Thats exactly how I wrote it from the start, but covr could not handle the difference between "integer" and "double" so I gave up and made this work around